homeassistant/components/airq/sensor.py

```diff
@@ -102,14 +102,13 @@ class AirQEntityDescription(SensorEntityDescription, AirQEntityDescriptionMixin)
     ),
     AirQEntityDescription(
         key="co",
-        translation_key="carbon_monoxide",
+        device_class=SensorDeviceClass.CO,
```

Comment:
This device class isn't valid for the unit of measurement provided by this sensor

Reply:
Don't forget to put back translations in this case 😉

Reply:
Oops :P